graph.py

```diff
@@ -9,12 +9,47 @@
     PACKAGE = 'networkx>=2.2'
     log.error(f"Package {PACKAGE} not found. Please 'pip install {PACKAGE}'")
 
+from itertools import combinations
+
+class Filter:
```

Comment:
Filter class contains the type of values expected in a list and the filter function that will filter out items from that list.